packages/asc-ui/src/components/TextArea/TextAreaMethodsContext.ts

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const TextAreaMethodsContext = React.createContext({})
```

Comment:
Do we really need the context? I think we can remove it. I know we did the same for the Input component to retrieve the Input's ref in parent components, but I think that's a design flaw that we should revise. The user should compose the Input component together with other components, not like we did on SearchBar or sth. @jonkoops what's your view on this?

Reply:
Yeah, I totally agree with this. We've made the `Input` too complex in this way and I think we should refactor this in the future as well. I'd suggest to keep the `TextArea` as simple as possible.

Reply:
@jonkoops @chrisvanmook 
Agreed, I'll remove the `TextAreaMethodsContext `